src/test/scala/uk/gov/hmrc/play/views/helpers/FormWithCSRFSpec.scala

```diff
@@ -16,18 +16,18 @@
 
 package uk.gov.hmrc.play.views
 package helpers
-import uk.gov.hmrc.play.MessagesSupport
-import play.twirl.api.Html
-import play.api.test.Helpers._
 import org.scalatest.{Matchers, WordSpec}
 import play.api.mvc.Call
 import play.api.test.FakeRequest
-import play.twirl.api.HtmlFormat
-import uk.gov.hmrc.play.views.html.helpers.form
+import play.api.test.Helpers._
+import play.twirl.api.{Html, HtmlFormat}
+import uk.gov.hmrc.play.MessagesSupport
+import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
 
 class FormWithCSRFSpec extends WordSpec with Matchers with MessagesSupport with CSRFSpec {
 
   "@helpers.formWithCSRF" should {
+    val form       = new FormWithCSRF()
```

Comment:
Is this `val form` used?

Reply:
yup, but it's a bit hidden because it's just used to generate the the formElements we assert on, for example https://github.com/hmrc/play-ui/blob/4dd06488763483317328ac7f582091179b7af552/src/test/scala/uk/gov/hmrc/play/views/helpers/FormWithCSRFSpec.scala#L35